.github/workflows/ci.yaml

```diff
@@ -354,5 +349,16 @@ jobs:
           build-args: |
             SAML_IDP_METADATA_PATH=./config/idp-metadata-staging.xml
           push: true
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
+          tags: |
+            ${{ env.IMAGE }}:sha-${{ env.GITHUB_SHA_SHORT }}, \
+            ${{ env.IMAGE }}:main, \
+            ${{ env.IMAGE }}:latest
+
+      - name: Deploy to Staging via Webhook
+        uses: navied/secure-actions-webhook@0.2.0
```

Comment:
If I am correct this was written by senior staff engineer @ shopify, last updated a few years ago, one star on the repo. How did you find this? 👀

Reply:
https://github.com/marketplace?type=actions. I needed an HMAC SHA256 webhook action, and there are precious few for some reason.